### ISSUE TITLE
headless mode fail, But gui mode works fine

### ISSUE THREAD

**Issue by MacBook-Pro-gala** [windingwind/seal-3d#4]
When I don't use gui mode, it seems to fail during the test phase
CUDA11.6、gcc10.3、torch1.12.1 ubuntu18.04

Loading model from: /home/user/.conda/envs/torch-ngp/lib/python3.8/site-packages/lpips/weights/v0.1/alex.pth
[INFO] Trainer: ngp | 2023-08-29_11-03-50 | cuda | fp16 | exps/lego_ngp
[INFO] #parameters: 24490848
[INFO] Loading exps/lego_ngp/checkpoints/ngp_ep0150.pth ...
[INFO] loaded model.
[INFO] load at epoch 150, global step 30000
[INFO] loaded optimizer.
[INFO] loaded scheduler.
[INFO] loaded scaler.
[INFO] Trainer: ngp | 2023-08-29_11-03-50 | cuda | fp16 | exps/lego_brush_compare
[INFO] #parameters: 24490848
[INFO] Loading exps/lego_ngp/checkpoints/ngp_ep0150.pth ...
[INFO] loaded model.
[INFO] load at epoch 150, global step 30000
[INFO] loaded optimizer.
[INFO] loaded scheduler.
[INFO] loaded scaler.
Local x generation: 0.873894214630127
Surrounding x generation: 1.1920928955078125e-06
Global x generation: 4.76837158203125e-07
Loading train data: 100%|█████████████████████████████████████████| 100/100 [00:01<00:00, 59.49it/s]
Loading val data: 100%|███████████████████████████████████████████| 100/100 [00:01<00:00, 59.85it/s]
[INFO] Proxy train/eval/test: True/True/False
Proxying train data:   0%|                                                  | 0/100 [00:00<?, ?it/s]段错误 (核心已转储)

**Comment by MacBook-Pro-gala** [comment]
This is my conda list:
# packages in environment at /home/user/.conda/envs/torch-ngp:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    https://repo.anaconda.com/pkgs/main
_openmp_mutex             5.1                       1_gnu    https://repo.anaconda.com/pkgs/main
addict                    2.4.0                    pypi_0    pypi
ansi2html                 1.8.0                    pypi_0    pypi
asttokens                 2.2.1                    pypi_0    pypi
attrs                     23.1.0                   pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
ca-certificates           2023.05.30           h06a4308_0    https://repo.anaconda.com/pkgs/main
certifi                   2023.7.22                pypi_0    pypi
charset-normalizer        3.2.0                    pypi_0    pypi
click                     8.1.7                    pypi_0    pypi
clip                      0.2.0                    pypi_0    pypi
cmake                     3.27.2                   pypi_0    pypi
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
comm                      0.1.4                    pypi_0    pypi
configargparse            1.7                      pypi_0    pypi
contourpy                 1.1.0                    pypi_0    pypi
cycler                    0.11.0                   pypi_0    pypi
dash                      2.12.1                   pypi_0    pypi
dash-core-components      2.0.0                    pypi_0    pypi
dash-html-components      2.0.0                    pypi_0    pypi
dash-table                5.0.0                    pypi_0    pypi
dearpygui                 1.9.1                    pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
executing                 1.2.0                    pypi_0    pypi
fastjsonschema            2.18.0                   pypi_0    pypi
filelock                  3.12.2                   pypi_0    pypi
flask                     2.2.5                    pypi_0    pypi
fonttools                 4.42.1                   pypi_0    pypi
freetype                  2.10.4               h0708190_1    conda-forge
fvcore                    0.1.5.post20210915            py38    fvcore
giflib                    5.2.1                h36c2ea0_2    conda-forge
idna                      3.4                      pypi_0    pypi
imageio                   2.31.1                   pypi_0    pypi
imageio-ffmpeg            0.4.8                    pypi_0    pypi
importlib-metadata        6.8.0                    pypi_0    pypi
importlib-resources       6.0.1                    pypi_0    pypi
iopath                    0.1.9                      py38    iopath
ipython                   8.12.2                   pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
ipywidgets                8.1.0                    pypi_0    pypi
itsdangerous              2.1.2                    pypi_0    pypi
jedi                      0.19.0                   pypi_0    pypi
jinja2                    3.1.2                    pypi_0    pypi
joblib                    1.3.2                    pypi_0    pypi
jpeg                      9e                   h166bdaf_1    conda-forge
json5                     0.9.14                   pypi_0    pypi
jsonschema                4.19.0                   pypi_0    pypi
jsonschema-specifications 2023.7.1                 pypi_0    pypi
jupyter-core              5.3.1                    pypi_0    pypi
jupyterlab-widgets        3.0.8                    pypi_0    pypi
kiwisolver                1.4.5                    pypi_0    pypi
lcms2                     2.12                 h3be6417_0    defaults
ld_impl_linux-64          2.38                 h1181459_1    https://repo.anaconda.com/pkgs/main
lerc                      3.0                  h295c915_0    defaults
libdeflate                1.17                 h5eee18b_0    defaults
libffi                    3.4.4                h6a678d5_0    https://repo.anaconda.com/pkgs/main
libgcc-ng                 11.2.0               h1234567_1    https://repo.anaconda.com/pkgs/main
libgomp                   11.2.0               h1234567_1    https://repo.anaconda.com/pkgs/main
libpng                    1.6.39               h5eee18b_0    defaults
libstdcxx-ng              11.2.0               h1234567_1    https://repo.anaconda.com/pkgs/main
libtiff                   4.5.0                h6a678d5_2    defaults
libwebp                   1.2.4                h11a3e52_1    defaults
libwebp-base              1.2.4                h5eee18b_1    defaults
lit                       16.0.6                   pypi_0    pypi
lpips                     0.1.4                    pypi_0    pypi
lz4-c                     1.9.4                h6a678d5_0    defaults
markdown-it-py            3.0.0                    pypi_0    pypi
markupsafe                2.1.3                    pypi_0    pypi
matplotlib                3.7.2                    pypi_0    pypi
matplotlib-inline         0.1.6                    pypi_0    pypi
mdurl                     0.1.2                    pypi_0    pypi
mpmath                    1.3.0                    pypi_0    pypi
msgpack                   1.0.5                    pypi_0    pypi
nbformat                  5.7.0                    pypi_0    pypi
ncurses                   6.4                  h6a678d5_0    https://repo.anaconda.com/pkgs/main
nest-asyncio              1.5.7                    pypi_0    pypi
networkx                  3.1                      pypi_0    pypi
ninja                     1.11.1                   pypi_0    pypi
numpy                     1.24.4                   pypi_0    pypi
nvidia-cublas-cu11        11.10.3.66               pypi_0    pypi
nvidia-cuda-cupti-cu11    11.7.101                 pypi_0    pypi
nvidia-cuda-nvrtc-cu11    11.7.99                  pypi_0    pypi
nvidia-cuda-runtime-cu11  11.7.99                  pypi_0    pypi
nvidia-cudnn-cu11         8.5.0.96                 pypi_0    pypi
nvidia-cufft-cu11         10.9.0.58                pypi_0    pypi
nvidia-curand-cu11        10.2.10.91               pypi_0    pypi
nvidia-cusolver-cu11      11.4.0.1                 pypi_0    pypi
nvidia-cusparse-cu11      11.7.4.91                pypi_0    pypi
nvidia-nccl-cu11          2.14.3                   pypi_0    pypi
nvidia-nvtx-cu11          11.7.91                  pypi_0    pypi
open3d                    0.17.0                   pypi_0    pypi
opencv-python             4.8.0.76                 pypi_0    pypi
openssl                   3.0.10               h7f8727e_2    https://repo.anaconda.com/pkgs/main
packaging                 23.1                     pypi_0    pypi
pandas                    2.0.3                    pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
pexpect                   4.8.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pillow                    10.0.0                   pypi_0    pypi
pip                       23.2.1           py38h06a4308_0    https://repo.anaconda.com/pkgs/main
pkgutil-resolve-name      1.3.10                   pypi_0    pypi
platformdirs              3.10.0                   pypi_0    pypi
plotly                    5.16.1                   pypi_0    pypi
portalocker               1.4.0                      py_0    conda-forge
prompt-toolkit            3.0.39                   pypi_0    pypi
protobuf                  4.24.1                   pypi_0    pypi
ptyprocess                0.7.0                    pypi_0    pypi
pure-eval                 0.2.2                    pypi_0    pypi
pygments                  2.16.1                   pypi_0    pypi
pymcubes                  0.1.4                    pypi_0    pypi
pyparsing                 3.0.9                    pypi_0    pypi
pyquaternion              0.9.9                    pypi_0    pypi
pysdf                     0.1.9                    pypi_0    pypi
python                    3.8.17               h955ad1f_0    https://repo.anaconda.com/pkgs/main
python-dateutil           2.8.2                    pypi_0    pypi
pytorch3d                 0.7.4           py38_cu116_pyt1121    <unknown>
pytz                      2023.3                   pypi_0    pypi
pyyaml                    6.0.1                    pypi_0    pypi
pyzmq                     25.1.1                   pypi_0    pypi
qtpy                      2.3.1                    pypi_0    pypi
readline                  8.2                  h5eee18b_0    https://repo.anaconda.com/pkgs/main
referencing               0.30.2                   pypi_0    pypi
requests                  2.31.0                   pypi_0    pypi
retrying                  1.3.4                    pypi_0    pypi
rfc3339-validator         0.1.4                    pypi_0    pypi
rfc3986-validator         0.1.1                    pypi_0    pypi
rich                      13.5.2                   pypi_0    pypi
rpds-py                   0.9.2                    pypi_0    pypi
scikit-learn              1.3.0                    pypi_0    pypi
scikit-spatial            7.0.0                    pypi_0    pypi
scipy                     1.10.1                   pypi_0    pypi
send2trash                1.8.2                    pypi_0    pypi
setuptools                68.0.0           py38h06a4308_0    https://repo.anaconda.com/pkgs/main
six                       1.16.0                   pypi_0    pypi
sniffio                   1.3.0                    pypi_0    pypi
soupsieve                 2.4.1                    pypi_0    pypi
sqlite                    3.41.2               h5eee18b_0    https://repo.anaconda.com/pkgs/main
stack-data                0.6.2                    pypi_0    pypi
sympy                     1.12                     pypi_0    pypi
tabulate                  0.9.0              pyhd8ed1ab_1    conda-forge
tenacity                  8.2.3                    pypi_0    pypi
tensorboardx              2.6.2.2                  pypi_0    pypi
termcolor                 2.3.0              pyhd8ed1ab_0    conda-forge
terminado                 0.17.1                   pypi_0    pypi
threadpoolctl             3.2.0                    pypi_0    pypi
tinycss2                  1.2.1                    pypi_0    pypi
tk                        8.6.12               h1ccaba5_0    https://repo.anaconda.com/pkgs/main
tomli                     2.0.1                    pypi_0    pypi
torch                     1.12.1+cu116             pypi_0    pypi
torch-ema                 0.3                      pypi_0    pypi
torchaudio                0.12.1+cu116             pypi_0    pypi
torchvision               0.13.1+cu116             pypi_0    pypi
tornado                   6.3.3                    pypi_0    pypi
tqdm                      4.66.1             pyhd8ed1ab_0    conda-forge
traitlets                 5.9.0                    pypi_0    pypi
trimesh                   3.23.5                   pypi_0    pypi
triton                    2.0.0                    pypi_0    pypi
typing-extensions         4.7.1                    pypi_0    pypi
tzdata                    2023.3                   pypi_0    pypi
uri-template              1.3.0                    pypi_0    pypi
urllib3                   2.0.4                    pypi_0    pypi
wcwidth                   0.2.6                    pypi_0    pypi
webcolors                 1.13                     pypi_0    pypi
webencodings              0.5.1                    pypi_0    pypi
websocket-client          1.6.2                    pypi_0    pypi
werkzeug                  2.2.3                    pypi_0    pypi
wheel                     0.38.4           py38h06a4308_0    https://repo.anaconda.com/pkgs/main
widgetsnbextension        4.0.8                    pypi_0    pypi
xz                        5.4.2                h5eee18b_0    https://repo.anaconda.com/pkgs/main
yacs                      0.1.8              pyhd8ed1ab_0    conda-forge
yaml                      0.2.5                h7f98852_2    conda-forge
zipp                      3.16.2                   pypi_0    pypi
zlib                      1.2.13               h5eee18b_0    https://repo.anaconda.com/pkgs/main
zstd                      1.5.5                hc292b87_0    defaults

**Comment by windingwind** [comment]
Try https://github.com/jwyang/faster-rcnn.pytorch/issues/816#issuecomment-1062691308

**Comment by windingwind** [comment]
Since I don't get further replies, I will close this issue. Please feel free to open it again.
